database/cassandra/cassandra.go

```diff
@@ -72,18 +79,8 @@ func (c *Cassandra) Open(url string) (database.Driver, error) {
 		return nil, ErrNoKeyspace
 	}
 
-	migrationsTable := u.Query().Get("x-migrations-table")
-	if len(migrationsTable) == 0 {
```

Comment:
why remove the fallback to the default migrations table?

Reply:
It happens within `WithInstance` so this is duplicating the logic since Open calls `WithInstance`.